integreat_cms/cms/views/organizations/organization_form_view.py

```diff
@@ -0,0 +1,140 @@
+            id=kwargs.get("organization_id")
+        ).first()
+
+        if organization_instance and organization_instance.archived:
+            disabled = True
+            messages.warning(
+                request, _("You cannot edit this organization because it is archived.")
+            )
+        elif not request.user.has_perm("cms.change_organization"):
+            disabled = True
+            messages.warning(
+                request, _("You don't have the permission to edit organizations.")
+            )
+        else:
+            disabled = False
+
+        organization_form = OrganizationForm(
+            instance=organization_instance,
+            disabled=disabled,
+        )
+
+        return render(
+            request,
+            self.template_name,
+            {
+                **self.get_context_data(**kwargs),
+                "form": organization_form,
+                "current_menu_item": "organizations",
+            },
+        )
+
+    def post(self, request: HttpRequest, **kwargs: Any) -> HttpResponse:
+        r"""
+        Save organization form for HTTP POST requests
+
+        :param request: Object representing the user call
+        :param \**kwargs: The supplied keyword arguments
+        :return: The rendered template response
+        """
+        region = request.region
+
+        organization_instance = Organization.objects.filter(
+            id=kwargs.get("organization_id")
+        ).first()
+
+        if organization_instance and organization_instance.archived:
+            return redirect(
+                "edit_organization",
+                **{
+                    "region_slug": region.slug,
+                    "organization_id": organization_instance.id,
+                },
+            )
+
+        organization_form = OrganizationForm(
+            data=request.POST,
+            files=request.FILES,
+            instance=organization_instance,
+            additional_instance_attributes={
+                "region": region,
+            },
+        )
+
+        if organization_form.is_valid():
+            organization_form.save()
+
+            if not organization_instance:
+                messages.success(
+                    request,
+                    _('Organization "{}" was successfully created').format(
+                        organization_form.instance
+                    ),
+                )
+            else:
+                messages.success(
+                    request,
+                    _('Organization "{}" was successfully saved').format(
+                        organization_form.instance
+                    ),
+                )
+        else:
+            organization_form.add_error_messages(request)
+
+        return render(
+            request,
+            self.template_name,
+            {
+                **self.get_context_data(**kwargs),
+                "organization_form": organization_form,
```

Comment:
```suggestion
                "form": organization_form,
```
We missed that one yesterday